roles/passenger/README.md

```diff
@@ -0,0 +1,57 @@
+# Passenger
+
+Borrowed from [pulibrary / princeton_ansible](https://github.com/pulibrary/princeton_ansible/tree/main/roles/passenger)
+
+Originally borrowed from Geerling Guy
+
+Installs Passenger (with Nginx) Ubuntu linux servers.
+
+## Requirements
+
+None.
+
+## Role Variables
+
+Available variables are listed below, along with default values (see `defaults/main.yml`):
+
+    passenger_server_name: www.example.com
+
+The server name (used in the Nginx virtual host configuration).
+
+    passenger_app_root: /opt/example/public
+
+The `passenger_root` for your application (e.g. the `public` folder in a rails app).
+
+    passenger_app_env: production
+
+The app environment passenger will load.
+
+    passenger_root: /usr/lib/ruby/vendor_ruby/phusion_passenger/locations.ini
+    passenger_ruby: /usr/local/bin/ruby
+
+Values for passenger configuration directives inside `nginx.conf`. These defaults should generally work correctly, but if you build `ruby` on its own (as an example), the path to ruby may be different.
```

Comment:
might be worth updating this to talk about python instead since we won't be building/configuring ruby